mercury/grafanaAPI/grafana_api.py

```diff
@@ -102,6 +117,18 @@ def get_dashboard_by_event_name(self, event_name):
         else:
             return None
 
+    def get_dashboard_url_by_name(self, name):
+        name = name.lower().replace(" ", "-")
```

Comment:
Optional: Is converting "Mercury Dashboard   " into "mercury-dashboard---" expected behavior? (Note that the trailing whitespace)
If not, we should have strip() here. Otherwise, ignore this comment.
```
name.strip().lower().replace(" ", "-")
```

Reply:
awesome catch we will fix that